lib/omnibus/signing.rb

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Omnibus
+  module Signing
```

Comment:
Perhaps we should instead create a `Omnibus::Packager::WindowsBase ` (which extends from `Omnibus::Packager::Base`) class and place these shared methods in it? That would more idiomatically follow how the rest of Omnibus code is laid out. A global module like this would make more sense if we needed to mix in these methods into non-packaging code.